java-frontend/src/main/java/org/sonar/java/cfg/CFG.java

```diff
@@ -700,6 +700,15 @@ private void buildBreakStatement(BreakStatementTree tree) {
       targetBlock = labelsBreakTarget.get(label.name());
     }
     currentBlock = createUnconditionalJump(tree, targetBlock);
+    if(currentBlock.exitBlock != null) {
+      currentBlock.exitBlock = null;
+      Block last = breakTargets.removeLast();
+      targetBlock.successors().clear();
```

Comment:
The NPE reported here is a FP caused by learning the NULL constraint from method invocation `createUnconditionalJump(Tree, @Nullable Block)` [line 702] which makes the engine learn that target block may be nullable.